README.md

```diff
@@ -21,10 +21,55 @@ other components of OpenShift via support for a baremetal platform type.
 
 # Instructions
 
-## Configuration
+## Preparation
+
+Considering that this is a new install on a clean OS, the next tasks should be performed prior the installation:
+
+1. Enable passwordless sudo for the current user
```

Comment:
for security reasons, a different user should be created, making sure that it doesn't have have access from external

Reply:
@elfosardo Do you mean that there should be two non-root users and one of them has no SSH access?

Reply:
Any user will work - whether you decide to create a new special user is personal preference IMO, the main thing is not to run dev-scripts as root, otherwise it's likely any future attempt to run as non-root will fail, due to various files then being owned by root.

Reply:
Oh of course.
Under "current user" I meant a personal, non-root user.
The GUI installation wizard for RHEL and Centos now makes it mandatory to create such a user, so I treat it as an obvious fact.

Reply:
I would never give passwordless sudo access to a non-root user with ssh access, it's basically like having a second root user, I usually create a user specifically for deployments, it takes maybe 10 seconds and can be scripted.
I understand this might be viewed as a personal preference, from my experience I see it as a security best practice.

Reply:
Creating a user for just the deployment is also an option. Depends on what most people prefer.
Still, how important is security in the dev-scripts cluster? Because a passwordless user is generally not secure.